### ISSUE TITLE
Missing "types" in package.json

### ISSUE THREAD

**Issue by Arnesfield** [Septh/rollup-plugin-node-externals#24]
Recently updated to [v6.1.0](https://github.com/Septh/rollup-plugin-node-externals/releases/tag/v6.1.0) and got an error (missing type declarations) for importing `rollup-plugin-node-externals` in `rollup.config.ts`.

Adding `types` to `package.json` should fix the issue:

```
"types": "./dist/index.d.ts"
```

Looks like `types` was removed in 8866dcfa2a7a652087905587fb8bc64a675ec287.

**Comment by Septh** [comment]
You're damn right. I'm not sure when I fucked this up - can't remember removing the line at all.

Moreover, `tsc --traceResolution` doesn't show any error in my local test:

```
======== Resolving module 'rollup-plugin-node-externals' from 'D:/Users/Stephan/Dev/zz_tests/node-externals/rollup.config.mjs'. ========
Explicitly specified module resolution kind: 'Node16'.
Resolving in ESM mode with conditions 'import', 'types', 'node'.
File 'D:/Users/Stephan/Dev/zz_tests/node-externals/package.json' exists according to earlier cached lookups.
Loading module 'rollup-plugin-node-externals' from 'node_modules' folder, target file types: TypeScript, JavaScript, Declaration.
Found 'package.json' at 'D:/Users/Stephan/Dev/zz_tests/node-externals/node_modules/rollup-plugin-node-externals/package.json'.
Using 'exports' subpath '.' with target './dist/index.js'.
File name 'D:/Users/Stephan/Dev/zz_tests/node-externals/node_modules/rollup-plugin-node-externals/dist/index.js' has a '.js' extension - stripping it.
File 'D:/Users/Stephan/Dev/zz_tests/node-externals/node_modules/rollup-plugin-node-externals/dist/index.ts' does not exist.
File 'D:/Users/Stephan/Dev/zz_tests/node-externals/node_modules/rollup-plugin-node-externals/dist/index.tsx' does not exist.
File 'D:/Users/Stephan/Dev/zz_tests/node-externals/node_modules/rollup-plugin-node-externals/dist/index.d.ts' exists - use it as a name resolution result.
Resolving real path for 'D:/Users/Stephan/Dev/zz_tests/node-externals/node_modules/rollup-plugin-node-externals/dist/index.d.ts', result 'D:/Users/Stephan/Dev/zz_tests/node-externals/node_modules/rollup-plugin-node-externals/dist/index.d.ts'.
======== Module name 'rollup-plugin-node-externals' was successfully resolved to 'D:/Users/Stephan/Dev/zz_tests/node-externals/node_modules/rollup-plugin-node-externals/dist/index.d.ts' with Package ID 'rollup-plugin-node-externals/dist/index.d.ts@6.1.0'. ========
```

I'll accept your PR but meanwhile could you please share your `tsconfig.json`? I'm curious to compare it with mine. Thanks. 🙏

**Comment by Septh** [comment]
v6.1.1 is out, thanks again.

**Comment by Arnesfield** [comment]
My `tsconfig.json` had `moduleResolution` set to `node`. Tried changing it to `node16` or `nodenext` and it actually fixes the types issue for [v6.1.0](https://github.com/Septh/rollup-plugin-node-externals/releases/tag/v6.1.0).

My PR was probably unnecessary if the intention was to drop support for the older implementation, so It looks like it was a problem on my end all along lol

**Comment by Septh** [comment]
> My PR was probably unnecessary if the intention was to drop support for the older implementation, so It looks like it was a problem on my end all along lol

Not at all. It was a stupid mistake on my side. Thanks again for the help.
